Dropdown/Dropdown.module.less

```diff
@@ -175,6 +175,8 @@
 				border-width: @agate-dropdown-item-border-width;
 				padding-left: @agate-dropdown-item-padding;
 				padding-right: @agate-dropdown-item-padding;
+				height: @agate-dropdown-item-height;
+				max-width: @agate-dropdown-item-max-width;
```

Comment:
I don't think it's needed.

Reply:
The `max-width` property could be removed since the item doesn't have the max-width anymore, but we still need the `height` property because now small item (in dropdown's case the radioItem) has a height of 99px and the layout is broken as seen below
![dropdown](https://user-images.githubusercontent.com/63335823/113114575-243e5c80-9214-11eb-80e1-5f2e13d7f07f.PNG)

Reply:
I meant "max-width". 👌